suites/portal/studyViewerTest.js

```diff
@@ -5,15 +5,44 @@ const { Bash } = require('../../utils/bash.js');
 const studyViewerTasks = require('../../services/portal/studyViewer/studyViewerTasks.js');
```

Comment:
Please add some code comments to help others understand the pre-requisites around this test.